docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -304,6 +304,146 @@ options were created in the previous steps.
  The last two lines also confirm that the correct devices are being used for
 the `Data file` and the `Metadata file`.
 
+### Extend devicemapper thin-pool
```

Comment:
Thanks, refreshed the patch with your gist and changed a typo and a format issue